zend_abstract_interface/sandbox/sandbox.h

```diff
@@ -356,8 +375,25 @@ inline bool zai_sandbox_timed_out(void) {
     return false;
 }
 
+inline bool zai_is_request_blocked(void)
```

Comment:
Both implementations of this function are exactly the same, perhaps you can define it only once?

Reply:
They are the same yes. However, this file is full of this duplication. I did it that way so follow the same approach

Reply:
Unless there's a good technical reason, don't duplicate the function.

Reply:
I get your point but have a look to the whole file. It done using this approach and I will follow it. In fact now the functions aren't the same anymore